### DISCUSSION TITLE
Flexibility in Taskbar

### DISCUSSION THREAD

**Discussion by sarkar-sayantan** [React95/React95#368] [Ideas]
Hi guys,

I think it would be more flexible if you guys support sending text and image as props for Taskbar.

**Comment by ggdaltoso** [comment]
Could you elaborate more?

Currently, the TaskBar will show:
 - The "Start" Button
 - A button for each modal opened
 - the date on the right side, on hover

![image](https://user-images.githubusercontent.com/6536985/182596317-13523883-4f52-45c9-a2db-8235ec06c719.png)

How would your changes look like?

**Comment by sarkar-sayantan** [comment]
I was talking about Start Button and its logo. It should be customizable i.e. If I pass "XYX" and "logo.png" as props that button name should be changed to "XYZ" with new logo.

**Reply by ggdaltoso** [reply]
Would you wanna contribute with this feature?

**Reply by sarkar-sayantan** [reply]
Yeah sure.
